pcdsdevices/epics_motor.py

```diff
@@ -735,6 +737,27 @@ def flag_is_cleared(value=None, **kwargs):
             status_wait(st, timeout=timeout)
         return st
 
+    @property
+    def md(self):
+        if self._md is None:
+            raise AttributeError('Device does not have an attached md, '
+                                 'and was likely not initialized from happi')
+        return self._md
+
+    @md.setter
+    def md(self, new_md):
+        """ initialize attributes when md is set """
+        self._md = new_md
+        self._extra = self._md.extraneous
+        self._id = self._extra['_id']
+        self._stageidentity = self._extra['stageidentity']
+        self._pvbase = self._extra['pvbase']
+        if self._stageidentity == "NEW":
+            print(f"This is a new stage, parameter configuration does not exist yet. Please manually configure {self._id}.")
```

Comment:
Note that this print will end up being interleaved into the big loading block, so it might be easy to miss. Is there another method that could be more helpful to show this during? Or would that be too spammy?

Reply:
It was expected to print in the big block but instead of a plain print statement, I will do a "logger.warning(...)" to at least add some differentiation.

Reply:
You also have access to `self.log.warning(...)` for ophyd devices, which adds in some metadata about the calling device (I think?)